core/README.md

```diff
@@ -156,3 +204,113 @@ the [overview](../overview.md#example-landing-page) document.
+whether the Collection is used as part of the tree or not:
+
+- Catalog (root) -> Catalog* -> Item (recommended)
+- Catalog (root) -> Collection -> Catalog* -> Item
+
+All items must be part of a Collection, but the Collection itself does not need to be part of the browsable graph.
+
+How you structure your graph of Catalogs can allow you to both group Collections together and create sub-groups
+of items within a Collection. 
+For example, your collections may be grouped so each represent a data product. This might mean
+you have a collection for each of Landsat 8 Collection 1, Landsat 8 Surface Reflectance, Sentinel-2 L1C, Sentinel-2
+L2A, Sentinel-5P UV Aerosol Index, Sentinel-5P Cloud, MODIS MCD43A4, MODIS MOD11A1, and MODIS MYD11A1. You can also
+present each of these as a catalog, and create parent catalogs for them that allow you to group together all Landsat, Sentinel, and MODIS catalogs.
+
+- / root catalog
+  - child -> /catalogs/landsat
+    - child -> /catalogs/landsat_7
+    - child -> /catalogs/landsat_8
+      - child -> /catalogs/landsat_8_c1
+      - child -> /catalogs/landsat_8_sr
+  - child -> /catalogs/sentinel
+    - child -> /catalogs/sentinel_2
+      - child -> /catalogs/sentinel_2_l1c
+      - child -> /catalogs/sentinel_2_l2a
+    - child -> /catalogs/sentinel_5p
+      - child -> /catalogs/sentinel_5p_uvai
+      - child -> /catalogs/sentinel_5p_cloud
+  - child -> /catalogs/modis
+    - child -> /catalogs/modis_mcd43a4
+    - child -> /catalogs/modis_mod11a1 
+    - child -> /catalogs/modis_myd11a1
+
+Each of these catalog endpoints could in turn be its own STAC API root, allowing an interface where users can
+search over arbitrary groups of collections without needing to explicitly know and name every collection in the
+search `collection`  query parameter. These catalogs-of-catalogs can be separated multiple ways, e.g. be
+per provider (e.g., Sentinel-2), per domain (e.g., cloud data), or per form of data (electro-optical, LIDAR, SAR).
+
+Going the other direction, collections can be sub-grouped into smaller catalogs. For example, this example
+groups a catalog of Landsat 8 Collection 1 items by path, row, and date (the path/row system is used by this
+product for gridding). 
+
+- / (root)
+  - /catalogs/landsat_8_c1
+    - /catalogs/landsat_8_c1/139
+      - /catalogs/landsat_8_c1/139_045
+        - /catalogs/landsat_8_c1/139_045_20170304
+          - /collection/landsat_8_c1/items/LC08_L1TP_139045_20170304_20170316_01_T1
+        - /catalogs/landsat_8_c1/139_045_20170305
+          - /collection/landsat_8_c1/items/LC08_L1TP_139045_20170305_20170317_01_T1
+      - /catalogs/landsat_8_c1/139_046
+        - /catalogs/landsat_8_c1/139_046_20170304
+          - /collection/landsat_8_c1/items/LC08_L1TP_139046_20170304_20170316_01_T1
+        - /catalogs/landsat_8_c1/139_046_20170305
+          - /collection/landsat_8_c1/items/LC08_L1TP_139046_20170305_20170317_01_T1
```

Comment:
Are you avoiding adding the child catalog links to the Collection to avoid URLs like `/collections/landsat_8_c1/139_046_20170304`, which would be hard for servers to handle cases like say a sub-Catalog named "items"? The description `Catalog (root) -> Collection -> Catalog* -> Item` lends itself to the root Catalog having children that would return the Collection, which in turn has children Catalogs*, which contain Items. I don't think anything blocks that from being a proper layout, so why duplicate a Catalog and Collection both with ID "landsat_8_c1"?